doc/news/_preparation_next_release.md

```diff
@@ -41,7 +41,42 @@ From now on `keyNew ("/array/#10", KEY_END)` will create a `Key` with name `/arr
 The whole internal implementation for `keySetName`, `keyAddName`, etc. has been completely rewritten.
 If you rely on specific behaviour of Elektra's Keynames and have already taken the two changes above into account, please refer to the newly created [Keyname documentation](../keynames) and (easier to read) [Python reference implementation](../keynames/keynames.py).
```

Comment:
This "(easier to read)" implies the python implementation is easier to read than the docu.

```suggestion
If you rely on specific behaviour of Elektra's Keynames and have already taken the two changes above into account, please refer to the newly created [Keyname documentation](../keynames) and [Python reference implementation](../keynames/keynames.py).
```